scripts/travis/save-logs.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import sys
+import subprocess
+import mmap
+import os
+
+from datetime import datetime, timedelta
+
+def tail(filename, n):
```

Comment:
@caofangkun If the `tail` cmd can function well here, please use `tail`. It's easier to understand and maintain in the future.

For the example code you write in the reply, do we really need the `while` loop?